tsl/test/expected/continuous_aggs_policy.out

```diff
@@ -118,9 +121,10 @@ ERROR:  must be owner of continuous aggregate "mat_m1"
 SET ROLE :ROLE_DEFAULT_PERM_USER;
 DROP MATERIALIZED VIEW mat_m1;
 --- code coverage tests : add policy for timestamp and date based table ---
+--- to prevent NOTICES set message level to warning
+SET client_min_messages TO warning; 
```

Comment:
Can it be done at the place of the problematic test and then restored?

Reply:
done.